src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac_zh_CN.properties

```diff
@@ -187,7 +187,7 @@ javac.opt.Xdoclint.custom=为 javadoc 注释中的问题启用或禁用特定检
 
 javac.opt.Xdoclint.package.args = [-]<程序包>(,[-]<程序包>)*
 
-javac.opt.Xdoclint.package.desc=在特定的程序包中启用或禁用检查。每个 <程序包> 是\n程序包的限定名称，或程序包名称前缀后跟 '.*'\n（这将扩展到给定程序包的所有子程序包）。在每个 <程序包>\n前面加上 '-' 可以为指定程序包禁用检查。
+javac.opt.Xdoclint.package.desc=在特定的程序包中启用或禁用检查。每个 <package> 是\n限定的程序包名称，或程序包名称前缀后跟 ''.*''\n（这将扩展到给定程序包的所有子程序包）。在每个 <package>\n前面加上 ''-'' 可对指定程序包禁用检查。
```

Comment:
Looks like single-quotes are being changed to double single-quotes for some instances. Is this intentional? Or an issue with how the translation process reads this text?

Reply:
That's because the English source file had an update to introduce those double single quotes. (See https://cr.openjdk.org/~jlu/output/src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.html). It's just being updated to match.

Reply:
`每个 <package> 是\n限定的程序包名称.`
We use <package> here, but the args format is using Chinese translation,`= [-]<程序包>(,[-]<程序包>)*`

Should that be consistent?